Provision/base.sql

```diff
@@ -57,21 +50,21 @@ CREATE TABLE IF NOT EXISTS summoners
     created_at DATETIME DEFAULT NOW() NOT NULL,
     updated_at DATETIME DEFAULT NOW() NOT NULL,
     PRIMARY KEY (id),
-    FOREIGN KEY (player_icon_id) REFERENCES player_icon(id),
```

Comment:
We are storing it in the summoners table as an INT. we will use that variable to call upon the image in the future